src/jailer/src/env.rs

```diff
@@ -22,9 +22,21 @@ const STDIN_FILENO: libc::c_int = 0;
 const STDOUT_FILENO: libc::c_int = 1;
 const STDERR_FILENO: libc::c_int = 2;
 
+// Kernel-based virtual machine (hardware virtualization extensions)
+// minor/major numbers are taken from
+// https://www.kernel.org/doc/html/latest/admin-guide/devices.html
 const DEV_KVM_WITH_NUL: &[u8] = b"/dev/kvm\0";
+const DEV_KVM_MAJOR: u32 = 10;
+const DEV_KVM_MINOR: u32 = 232;
+
+// TUN/TAP device minor/major numbers are taken from
+// www.kernel.org/doc/Documentation/networking/tuntap.txt
 const DEV_NET_TUN_WITH_NUL: &[u8] = b"/dev/net/tun\0";
```

Comment:
```suggestion
const DEV_NET_TUN_WITH_NULL: &[u8] = b"/dev/net/tun\0";
```

Reply:
You sure you want this change in this PR?

Also, `NUL` is actually a thing: http://ascii-table.com/

Reply:
Was not aware of that, thanks :) you can ignore these suggestions.